docs/tables/aws_lambda_function.md

```diff
@@ -104,3 +104,16 @@ from
 where
   dead_letter_config_target_arn is null;
 ```
+
```

Comment:
```suggestion
```

Reply:
I have intentionally kept two line breaks because all the example queries in this doc have two lines

Reply:
@karanpopat I think early we had 2 line breaks in between examples, but in all newer plugins we're now using only 1. We don't need to go back and update all AWS table docs, but moving forward we should just use 1 and if we're touching a table doc that has 2 line breaks in between, we can adjust those as well

Reply:
@cbruno10 Got it. Removing the line breaks here then. Thanks!